apps/site/assets/ts/stop/components/LocationBlock.tsx

```diff
@@ -33,18 +32,14 @@ const addressOrMunicipality = (stop: Stop): ReactElement | null => {
   return null;
 };
 
+const formatMapLink = (location: string | null): string =>
+  location
+    ? `/trip-planner/to/${location.replace(/\s+/g, "-").toLowerCase()}`
+    : "";
+
```

Comment:
I think we need the encoder back! :smile: I found this works pretty well for stations so far, except we have a few stations with a `/` in the name - Charles/MGH, Kendall/MIT, JFK/Umass - for those stations the link leads to a 404.

If we use the encoder (`window.encodeURIComponent()`) function on the name/address that fixes the issue, so `/trip-planner/to/charles/mgh-station` (doesn't work) becomes `/trip-planner/to/charles%2Fmgh-station` (which works!)

Reply:
Are there any other cases the encoder would handle? Would a replace `/` with `%2F` suffice?

Reply:
Not sure off the top of my head, but we can try to look at all the stops and see.. but there are hundreds (maybe 1000?) of those. Not to mention we don't know what future stops (rare, but does happen) might look like!